package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "systemic-azure-table-storage",
+  "version": "1.0.0",
+  "description": "Azure Table Storage wrapper",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "lint": "eslint --ext .ts .",
+    "lint:fix": "eslint --fix --ext .ts .",
+    "test": "jest --runInBand --detectOpenHandles --forceExit --reporters=default",
+    "test:watch": "jest --runInBand --detectOpenHandles --forceExit --reporters=default --watch",
+    "test:coverage": "jest --runInBand --detectOpenHandles --forceExit --reporters=default --coverage",
+    "build": "tsc",
+    "infra:up": "bash -c 'docker-compose --file docker/docker-compose.yml --project-name azure-table-storage-azurite up -d --force-recreate --build'",
+    "infra:down": "bash -c 'docker-compose --file docker/docker-compose.yml --project-name azure-table-storage-azurite'",
+    "prepublish": "npm run build",
+    "release": "standard-version",
+    "release:major": "npm run release -- --release-as major",
+    "release:minor": "npm run release -- --release-as minor",
+    "release:patch": "npm run release -- --release-as patch",
+    "release:alpha": "npm run release -- --prerelease alpha"
+  },
+  "dependencies": {
+    "@azure/data-tables": "^13.1.1",
+    "debug": "^4.3.4"
+  },
+  "devDependencies": {
+    "@commitlint/cli": "^16.3.0",
+    "@commitlint/config-conventional": "^16.2.4",
+    "@types/debug": "^4.1.7",
+    "@types/jest": "^27.5.1",
+    "@types/node": "^17.0.33",
+    "@typescript-eslint/eslint-plugin": "^5.23.0",
+    "@typescript-eslint/parser": "^5.23.0",
+    "eslint": "^8.15.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-airbnb-typescript": "^17.0.0",
+    "eslint-plugin-import": "^2.26.0",
+    "husky": "^8.0.1",
+    "jest": "^28.1.0",
+    "standard-version": "^9.5.0",
+    "ts-jest": "^28.0.2",
+    "typescript": "^4.6.4"
+  },
+  "author": "",
```

Comment:
I would suggest adding you as author, or perhaps as contributor, just in case more people are added in the future.

In the future, perhaps you can also add the allcontributors cli commands to add new contributors.

Reply:
Yep, I wanted to check with you about how to incorporate `allcontributors` properly. I'll add my name temporarily